cdc/model/sink.go

```diff
@@ -287,10 +315,16 @@ func (r *RowChangedEvent) HandleKeyColumns() []*Column {
 
 // Column represents a column value in row changed event
 type Column struct {
-	Name  string         `json:"name"`
-	Type  byte           `json:"type"`
-	Flag  ColumnFlagType `json:"flag"`
-	Value interface{}    `json:"value"`
+	Name  string         `json:"name" msg:"name"`
+	Type  byte           `json:"type" msg:"type"`
+	Flag  ColumnFlagType `json:"flag" msg:"-"`
+	Value interface{}    `json:"value" msg:"value"`
+}
+
+// RedoColumn stores Column change
+type RedoColumn struct {
+	Column *Column `msg:"column"`
+	Flag   uint64  `msg:"flag"`
```

Comment:
Better to add comment about why we redefine a `RedoColumn` for msgp encoding instead of reusing `Column`